### DISCUSSION TITLE
Disable Global Wither Sound

### DISCUSSION THREAD

**Discussion by hirata-estate** [PurpurMC/Purpur#549] [__Completed__] [answered]
Disable/limit the global sound that plays when a wither is spawned. This sound is incredibly annoying when someone is farming withers. There's a plugin for it here: https://www.spigotmc.org/resources/withersound.76730/; but Purpur integration would be great.

wither-spawn-sound: true/false
wither-spawn-sound-range: 96

**Comment by 12emin34** [comment]
Spigot already adds a config option for the radius, but i just made a PR that allows you to fully disable it

**Reply by hirata-estate** [reply]
Didn't know that, thank you!

**Comment by encode42** [comment, marked as answer]
Closed in https://github.com/pl3xgaming/Purpur/commit/0548d34515931f29f4848d9bd2901a090a2a9806
